offset_manager.go

```diff
@@ -233,7 +233,12 @@ func (om *offsetManager) mainLoop() {
 	}
 }
 
+// flushToBroker is ignored if auto-commit offsets is disabled
 func (om *offsetManager) flushToBroker() {
+	if !om.conf.Consumer.Offsets.AutoCommit.Enable {
```

Comment:
Glad this line is fixed, it actually blocks all calls to Commit(), however sometimes we do wish to commit manually.